### ISSUE TITLE
[bug] Truncated display in environment pane

### ISSUE THREAD

**Issue by etienne-s** [rstudio/rstudio#1703]
```r
a <- 1:1e6
```
![rstudio_environment_pane_bug](https://user-images.githubusercontent.com/22758616/32328130-3e4df786-bfd9-11e7-903f-a85a71c7b79c.PNG)

It should be `int`, not `nt`. (In the same way, `num` becomes `um`, and so on)

RStudio 1.1.383

```
R version 3.4.2 (2017-09-28)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

Matrix products: default

locale:
[1] LC_COLLATE=French_France.1252  LC_CTYPE=French_France.1252   
[3] LC_MONETARY=French_France.1252 LC_NUMERIC=C                  
[5] LC_TIME=French_France.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.4.2 tools_3.4.2    yaml_2.1.14  
```

**Comment by dfalty** [comment]
Also reproduced this on the latest 1.2.150 with Mac Desktop (10.12.2).

**Comment by kevinushey** [comment]
This code is the culprit: https://github.com/rstudio/rstudio/blob/1867b7d59a747f0bebda18a9e1f9f8da83c083a5/src/gwt/src/org/rstudio/studio/client/workbench/views/environment/view/EnvironmentObjectList.java#L426-L428

**Comment by dfalty** [comment]
Verified fixed 1.2.160.
